src/routing.js

```diff
@@ -44,21 +44,31 @@ export const queryStringMiddleware = (history, {reduxPathname, routes}, config =
         const result = next(action);
         const state = store.getState();
         const currentPath = reduxPathname(state);
-        const routeConfig = routes[currentPath];
+        const routesConfig = !_.isArray(routes[currentPath]) ? [routes[currentPath]] : routes[currentPath];
 
-        if (routeConfig) {
-            const {listen, select, serialize} = routeConfig;
-            const qsState = select(state);
+        if (!_.isEmpty(routesConfig)) {
+            let queryString = '';
+            routesConfig.map((routeConfig) => {
+                if (routeConfig) {
+                    const { listen, select, serialize } = routeConfig;
```

Comment:
no space inside `{...}` (eslint didn't complain?)

Reply:
That is what happens when one tries vs code :sweat_smile: